File: mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int n;
    do
    {
        n = get_int("Size: ");
    }
    while (n < 1);

    int leftCount = n - 1;
    int rightCount = 0;
    for (int i = 0; i < n; i++)
    {
        for (int j = 0; j < n; j++)
        {
            if (leftCount <= j)
            {
                printf("#");
            }
            else
            {
                printf(" ");
            }
        }
        printf("  ");
        for (int j = 0; j < n; j++)
        {
            if (rightCount >= j)
            {
                printf("#");
            }
        }
        leftCount--;
        rightCount++;
        printf("\n");
    }
}

File: sentimental-mario-more/mario.py
from cs50 import get_int

n = 0
while n < 1 or n > 8:
    n = get_int("Size: ")

leftCount = n - 1
rightCount = 0
for i in range(n):
    for j in range(n):
        if leftCount <= j:
            print("#", end="")
        else:
            print(" ", end="")
    print("  ", end="")
    for j in range(n):
        if rightCount >= j:
            print("#", end="")
    leftCount -= 1
    rightCount += 1
    print("")
